Wiki.md

```diff
@@ -0,0 +1,251 @@
+           └─11020 node index.js
+```
+6. Quickly test that the loadbalancer is up by navigating to the ip:port (127.0.0.1:3001). It should present you with the standard plex media page. 
+7. Configure Nginx to reverse proxy the loadbalancer FQDN. You can find examples of an Nginx configuration here: https://github.com/UnicornTranscoder/Nginx
+8. Configure Plex Media Server access address
+    * Under Settings -> Server -> Network
+    * Set Custom server access URLs to the address to access the UnicornLoadBalancer (ie. https://unicornloadbalancer.example.com)
+9. Disable the Plex Relay option
+    * Under Settings -> Server -> Network
+10. Disable outside access to the plex port
+    * All requests to the Plex Media Server should pass through the UnicornLoadBalancer.
+    * If a user reaches the server directly, the user will not be able to start a stream, since the FFMPEG transcoder built-in to Plex will eventually be replaced by Unicorn's Transcoder. 
+    * To ensure users always hit the loadbalancer, configure IPTables to drop traffic destined to 32400 from outside your plex machine (drop traffic coming in through the physical nic). You'll need to install iptables-persistent, a package that persists ip tables across reboots, and then set your iptables rule. You must specify your interface, or else 32400 will be dropped completely, even for localhost access, and the loadbalancer will not work. 
+        * `apt-get install -y iptables-persistent`
+        * `iptables -A INPUT -p tcp --dport 32400 -i [your-network-interface] -j DROP`
+
+-----
+
+#### Configure UnicornFFMPEG
+UnicornFFMPEG will replace the Plex built-in transcoder. To build UnicornFFMPEG, complete the following steps:
+1. `cd /opt`
+2. `git clone https://github.com/UnicornTranscoder/UnicornFFMPEG.git`
+3. `cd UnicornFFMPEG`
+4. `npm install`
+5. `LB_URL=UnicornLoadBalancer.example.com npm start`
+    * This will override the default localhost:port option with the FQDN of the loadbalancer. This environment variable is required for a working setup. 
+    * After this npm command finishes, a new Plex Transcoder binary will be saved to the ./bin folder of the repository. Copy the linux binary to your plex docker folder (or anywhere you want). In this example we're copying it to `/root/docker/plexv2/binaries/Plex Transcoder-linux`.
+
+In order to replace the built-in transcoder with this new one, we need to mount it into our plex container. Simply add the following mount to your plex container:
+```
+- "/root/docker/plexv2/binaries/Plex Transcoder-linux:/usr/lib/plexmediaserver/Plex Transcoder"
+```
+where `/usr/lib/plexmediaserver` is the binary location. Most mainstream plex containers use this path, such as the container released by LinuxServier.io. 
+
+With that new volume/file mount, restart plex to replace the built-in transcoder with the "fake" transcoder that interprets the transcode arguments.
```

Comment:
@drouarb We need to restart the container to accept the new volume mount. Otherwise, when plex does call the transcoder, it's still going to have the built in one. Up to this point, the container still has the original. We have not replaced it. So perhaps we can reword this to "restart/run the container with the new volume mount definition"

Reply:
Oh yes, didn't understood that you were talking about a docker setup